pkg/scanfetcher/nexpose.go

```diff
@@ -143,6 +143,26 @@ func (n *NexposeClient) makePagedNexposeScanRequest(page int) (nexposeScanRespon
 	return scanResp, nil
 }
 
+// CheckDependencies makes a call to the nexpose endppoint "/api/3".
+// Because asset producer endpoints vary user to user, we want to hit an endpoint
+// that is consistent for any Nexpose user
+func (n *NexposeClient) CheckDependencies(ctx context.Context) error {
+	u, _ := url.Parse(n.Endpoint.String())
+	u.Path = path.Join("/api/3")
+	req, _ := http.NewRequest(http.MethodGet, u.String(), http.NoBody)
+	res, err := n.Client.Do(req)
+	if err != nil {
+		return err
+	}
+	defer res.Body.Close()
+	fmt.Println("hello world!")
```

Comment:
Task: I'm assuming this a forgotten debug line and should be removed

Reply:
done